### ISSUE TITLE
404 Not Found (OpenURI::HTTPError)

### ISSUE THREAD

**Issue by gaetanm** [sstephenson/global_phone#39]
I have just installed you gem and when I try the `global_phone_dbgen` command I get this 404 error.

I saw in a previous issue that it could be simply due to a bad URL, what should I do?

**Comment by igorvish** [comment]
Gem in rubygems does not updates since 2013. And for some reason we need to install separate gem to generate databse. But generating script already exists in bin/global_phone_dbgen on master branch.
This how it can be called directly:

``` ruby
# Gemfile. Install gem from github
gem 'global_phone', :git => 'https://github.com/sstephenson/global_phone.git', :ref => 'dd0894061f58479884e6cfa2d00382542dc77d5a'
```

``` bash
# call script directly
bundle exec `bundle show global_phone`/bin/global_phone_dbgen > db/global_phone.json
```

**Comment by skatkov** [comment]
Ran into same issue, this solve it for me:
https://github.com/sstephenson/global_phone/issues/14#issuecomment-51804790

**Comment by jeremy** [comment]
Fixed by #36
